### ISSUE TITLE
Winforms/WPF doesn't use the current runtime dependency

### ISSUE THREAD

**Issue by TanayParikh** [dotnet/aspnetcore#33259]
Winforms/WPF Samples used the wrong shared runtime dependency (unsynchronized with the rest of the repo / build).

**Comment by javiercn** [comment]
MVC depends on Windows forms and WPF and needs to go after them on the build graph. The current issue is that they are being built in parallel and can get incompatible runtime versions.

**Comment by dougbu** [comment]
@TanayParikh @javiercn is there something actionable here❔ I'm also unsure why this issue is parked anywhere other than under `area-blazor` because the samples are not infrastructure.

It is, by the way, possible to ensure the samples build w/ the same framework versions as the rest of the repo. Is that your aim❔

**Comment by javiercn** [comment]
@dougbu We are not asking to change the samples.

This not only affects samples, it also affects our ability to correctly build Blazor Webview for Winforms/WPF, which is part of the product.

**Comment by Eilon** [comment]
@dougbu we have a meeting at 1pm to discuss.

**Comment by Eilon** [comment]
Here's the error you get:

```
System.TypeLoadException: 'Could not load type 'System.IO.File' from assembly 'System.Runtime, Version=6.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.'
```

**Comment by Eilon** [comment]
I met with @wtgodbe and @mkArtakMSFT and we think the best thing to do is to have ASP.NET Core pick up runtimes to be coherent with WPF and WinForms. This changes the build layering, which will be discussed with the appropriate authorities.

**Comment by Eilon** [comment]
To fill folks in: We looked at various options, and it seems that making this repo (AspNetCore) depend on WindowsDesktop (for WPF and WinForms) is untenable because it would greatly increase the build time of .NET Core end-to-end.

We are instead considering moving the WinForms and WPF BlazorWebView controls to another repo that will make it easier to have the correct dependencies.

**Comment by Eilon** [comment]
These projects are moved to the MAUI repo, thus avoiding the problem: https://github.com/dotnet/maui/pull/1199

**Comment by Eilon** [comment]
Re-opening until I remove these from the AspNetCore repo to complete the move.

For .NET 6 Preview 5 these packages will ship from the MAUI repo.

**Comment by Eilon** [comment]
And the code is now removed from the AspNetCore repo: https://github.com/dotnet/aspnetcore/pull/33410

Everything is where it should be:

* AspNetCore has base platform-independet types (e.g. WebView): https://github.com/dotnet/aspnetcore/tree/main/src/Components/WebView
* Maui has the .NET MAUI, WPF, and WinForms `BlazorWebView` controls (and the common WebView2 code): https://github.com/dotnet/maui/tree/main/src/BlazorWebView
